ext/nnstreamer/tensor_filter/tensor_filter_tensorflow.c

```diff
@@ -81,7 +81,8 @@ tf_loadModelFile (const GstTensorFilterProperties * prop, void **private_data)
     }
   }
 
-  tf_mem_optmz = nnsconf_get_value_bool (NNSCONF_VAL_TF_MEM_OPTMZ);
+  tf_mem_optmz = nnsconf_get_custom_value_bool ("tensorflow",
```

Comment:
The method `nnsconf_get_custom_value_bool()` reads `nnstreamer.ini.in` file and set the variables.
If we read the `mem_optimization` value only from `.ini` file, however, we can't set the option with the environment variable.